### DISCUSSION TITLE
New notes created outside of workspace directories

### DISCUSSION THREAD

**Discussion by pbardosi** [epwalsh/obsidian.nvim#378] [Q&A] [answered]
Hi,

is it intentional to create new notes always in the vault root or my config lacks something? I thought that if I select a workspace, than all notes created will be placed in the configured workspace directory. But it's not the case.

Can anyone help me?

Thanks in advance.

P.

**Comment by epwalsh** [comment]
Hey @pbardosi can you provide a little more detail about your current config and what you're trying to do?

**Reply by epwalsh** [reply, marked as answer]
I just added some more documentation about workspaces. See https://github.com/epwalsh/obsidian.nvim?tab=readme-ov-file#workspaces. Hopefully that clarifies things for you.

**Reply by pbardosi** [reply]
Thank you @epwalsh! I was dumb, asked without thinking. Now it's clear. :)
